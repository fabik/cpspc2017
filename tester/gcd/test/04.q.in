12
6 7 10 15 18 19 20 21 26 27 28 30
27 28 30 