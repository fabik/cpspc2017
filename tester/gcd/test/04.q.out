1
31
59
86
112
133
153
172
190
205
215
227
